src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -12,6 +12,11 @@ require "./settings"
 require "./i18n"
 require "./database"
 require "./initializers/**"
+require "amber_render_module"
```

Comment:
Is there anyway for someone to create a plugin without it being in the modular format?

Reply:
I think the danger with not using the modular format for plugins is that there could easily be conflicts with existing application artifacts.  By separating plugins into a specific plugins folder there is no possibility of any clashes between existing artifacts in src/controller, src/models etc.